src/_new-settings.scss

```diff
@@ -4,6 +4,27 @@
 // library namespace
 $fd-namespace: fd !default;
 
+$fd-breakpoints: (
+  s: 320px,    // phone     <600
+  m: 600px,    // tablet    600-1023
+  l: 1024px,   // desktop   1024-1439
+  xl: 1440px,  // desktop   1440-1919
+  xxl: 1920px, // display   >1920
+) !default;
```

Comment:
If you are bringing this maybe consider using fiori3 breakpoints:
<img width="601" alt="Screen Shot 2020-07-06 at 11 05 12 AM" src="https://user-images.githubusercontent.com/39598672/86608523-abf28580-bf78-11ea-86fb-771c419493d9.png">

Reply:
i will refactor them in another PR as this one is big enough